### DISCUSSION TITLE
what's wrong with the  display of the buttons?

### DISCUSSION THREAD

**Discussion by shenweizju** [logancyang/obsidian-copilot#587] [Q&A]
<img width="297" alt="image" src="https://github.com/user-attachments/assets/b37b92ae-142f-45ab-8618-9df4928030ab">

**Comment by logancyang** [comment]
Is this the newest version? And is it on mobile?

**Reply by shenweizju** [reply]
Yes it’s the newest version, both copilot and obsidian, macOS

**Reply by logancyang** [reply]
It looks like this for me
<img width="552" alt="image" src="https://github.com/user-attachments/assets/44c33c08-3b3e-4e68-b267-95c992a976f5">

Looks like the CSS didn't apply correctly for you somehow 🤔
